Marlin/src/HAL/HAL_STM32F1/timers.h

```diff
@@ -47,6 +47,21 @@ typedef uint16_t hal_timer_t;
 #define STEP_TIMER_CHAN 1 // Channel of the timer to use for compare and interrupts
 #define TEMP_TIMER_CHAN 1 // Channel of the timer to use for compare and interrupts
 
+/**
+ * Note: Some timers that can be already allocated elsewhere
+ * 
+ * FAN0 PWM:
+ *   When FAN_SOFT_PWM is not defined in Configuration.h, temperature.cpp
+ *   uses the timer that belongs to FAN_PIN to generate FAN0 PWM signal.
```

Comment:
Fan pins are named FAN_PIN, FAN1_PIN, etc. I'm not sure the best way to refer to them as group, perhaps FANx_PIN?

```suggestion
 *   uses the timer that belongs to FANx_PIN to generate FAN PWM signal.
```

Reply:
Thanks, I did not realize that each fans can actually take a different timer.